src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -167,16 +171,25 @@ def _compiler_tags(self):
   @classmethod
   def register_options(cls, register):
     super().register_options(register)
+
     register('--force-compiler-tag-prefix', default='use-compiler', metavar='<tag>',
       help='Always compile targets marked with this tag with rsc, unless the workflow is '
            'specified on the cli.')
+
     register('--workflow', type=cls.JvmCompileWorkflowType,
       default=cls.JvmCompileWorkflowType.zinc_only, metavar='<workflow>',
-      help='The workflow to use to compile JVM targets.', fingerprint=True)
+      help='The default workflow to use to compile JVM targets. This is overriden on a per-target basis with the force-compiler-tag-prefix tag.', fingerprint=True)
+
+    register('--workflow-override', type=cls.JvmCompileWorkflowType,
+      default=None, metavar='<workflow_override>',
+      help='The workflow to use to compile JVM targets, overriding the "workflow" option as well as any force-compiler-tag-prefix tags applied to targets. An example use case is to quickly turn off outlining workflows in case of errors.', fingerprint=True)
```

Comment:
you can specify `choices` for the options, e.g. https://github.com/pantsbuild/pants/blob/master/src/python/pants/backend/jvm/subsystems/jvm_platform.py#L59

so you won't need to the extra logic to check for it later.